prestoadmin/yarn_slider/server.py

```diff
@@ -117,7 +124,7 @@ def install(presto_yarn_package):
 
     package_install_command = \
         '%s package --install --package %s --name %s' % \
-        (get_slider_bin(conf), package_file, conf[APPNAME])
+        (get_slider_bin(conf), package_file, SLIDER_PKG_NAME)
```

Comment:
should this also have --replacepkg (just in case the package already exists? I think it is ok to assume here that replacing a package is OK.